### PR TITLE
Remove milliseconds from Zip files summary report time fields

### PR DESCRIPTION
**Pull request by aliveni**

### CODE REVIEW

**Comment by rkondratowicz** [comment]
that's json, what about csv?

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ReportsController.java [hunk 1]
```diff
@@ -88,11 +75,26 @@ public ResponseEntity downloadZipFilesSummary(
...
 
-        File csvFile = CsvWriter.writeZipFilesSummaryToCsv(summary);
+        File csvFile = CsvWriter.writeZipFilesSummaryToCsv(toZipFileSummaryResponse(summary));
         return ResponseEntity
             .ok()
             .header(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=zip-files-summary.csv")
             .body(Files.readAllBytes(csvFile.toPath()));
     }
 
+    private List<ZipFilesSummaryReportItem> toZipFileSummaryResponse(List<ZipFileSummaryResponse> summary) {
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ReportsController.java]: function name inconsisten with the actual return type. Maybe just wrap in `ZipFilesSummaryReportListResponse` below.

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ReportsController.java]: or, even better, make this PR about milliseconds only and leave it as it was

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ReportsController.java]: sounds better. will update the pr

**Pull request merged**